lib/spatial_features/has_spatial_features.rb

```diff
@@ -203,10 +205,9 @@ def intersects?(other)
 
     def bounds
```

Comment:
It seems like we can now replace this entire method with:

```suggestion
    def bounds
       aggregate_feature&.bounds
    end
```

Reply:
The key here is we don't want to load the aggregate feature if we can avoid it. Some are many megabytes and take a noticeably long time to load.

Reply:
This probably deserves a comment.